owlkettle/adw.nim

```diff
@@ -563,6 +565,119 @@ proc `valSwipe=`*(flap: Flap, swipe: bool) =
   flap.valSwipeToOpen = swipe
   flap.valSwipeToClose = swipe
 
+type PackType* = enum
+  PackStart, PackEnd
```

Comment:
Not used?

Reply:
Huh, they'll plop up again in a different PR, could've sworn I saw a necessity for them here as well. Oh well, removed here.